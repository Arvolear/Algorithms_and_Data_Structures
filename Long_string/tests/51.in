61367731112
100107939069605178151619650784494725410638790327