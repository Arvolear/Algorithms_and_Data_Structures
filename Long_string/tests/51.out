100107939069605178151619650784494725472006521439
100107939069605178151619650784494725349271059215
6143397087000010024811451896638212167782355068944282553624
1631279782641822662397857150624238079